Delete reference and link to embedded BBC talk video (#245)
dfe-analytical-services/analysts-guide: writing-visualising/writing.qmd

**Best**: “Was the substantial rise and fall in the number of homicides in the 2000s in England (Figure 1) observed across all age groups and types of weapons (Figure 2)?”
There have been no numbers presented yet, just a statement that establishes the purpose of the statistics. Introducing your topic is important especially when presenting a series of charts or tables.

> [Here is a session by Robert Cuffe from the BBC talking at the 2023 SRA annual conference](https://www.youtube.com/embed/WGTMRGQS7FM?controls=0) about highlighting key messages and how our publications are used by journalists and the media.

Reviewer: ```suggestion
Robert Cuffe from the BBC [led a session at the 2023 SRA annual conference](https://www.youtube.com/embed/WGTMRGQS7FM?controls=0) discussing highlighting key messages and how our publications are used by journalists and the media.
```

> You can also find an interview with him here [Office for Statistics Regulation - Making an impact: how can statisticians help journalists to answer key questions with data?](https://osr.statisticsauthority.gov.uk/blog/author/robert-cuffe/).

Reviewer: ```suggestion
You can also find an interview with Robert in the Office for Statistics Regulation blog: [Making an impact: how can statisticians help journalists to answer key questions with data?](https://osr.statisticsauthority.gov.uk/blog/author/robert-cuffe/).
```
